### PR TITLE
Bug 1328277 - Make our management commands compatible with Django 1.10

### PR DESCRIPTION
**Pull request by edmorley**
Django 1.10 removed support for optparse, so commands must use the built-in argparse support instead:
https://docs.djangoproject.com/en/1.10/releases/1.8/#extending-management-command-arguments-through-command-option-list

The remove_perf_signatures command has been converted to a LabelCommand (since it's slightly more appropriate given the expected arguments):
https://docs.djangoproject.com/en/1.9/howto/custom-management-commands/#django.core.management.LabelCommand

Note: The `LabelCommand.label` property isn't mentioned in the docs (yet, I'll submit an upstream PR), but can be seen here:
https://github.com/django/django/blob/1.9.12/django/core/management/base.py#L539

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/2056)

### CODE REVIEW

File: treeherder/model/management/commands/migrate_to_intermediate_jobs.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import time
```
Comment (wlach) [inline treeherder/model/management/commands/migrate_to_intermediate_jobs.py]: We should just delete this one.

File: treeherder/perf/management/commands/remove_duplicate_signatures.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import json
```
Comment (wlach) [inline treeherder/perf/management/commands/remove_duplicate_signatures.py]: We can just delete this also.

File: treeherder/perf/management/commands/remove_single_test_summaries.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import time
```
Comment (wlach) [inline treeherder/perf/management/commands/remove_single_test_summaries.py]: We can delete this too.

**Comment by edmorley** [comment]
> This looks fine, there's a few things in here which I'd lean towards just deleting as opposed to modifying, but up to you.

Agreed; done :-)

**Pull request merged**